flow/connectors/mysql/qvalue_convert.go

```diff
@@ -492,6 +507,39 @@ func QValueFromMysqlRowEvent(
 				return nil, err
 			}
 			return types.QValueTimestamp{Val: tm.UTC()}, nil
+		case types.QValueKindBoolean:
+			// integer types shouldn't get here, but try work with schema changes
+			return types.QValueBoolean{Val: strings.EqualFold(val, "true") || strings.EqualFold(val, "t") || strings.EqualFold(val, "on") || strings.EqualFold(val, "yes") || strings.EqualFold(val, "1")}, nil
+		case types.QValueKindInt8:
+			v, _ := strconv.ParseInt(val, 10, 8)
```

Comment:
Up to you, but I think a customer warning would be appropriate for these

Reply:
added logging. in future these can be customer facing alongside skew loss warnings